contracts/container/contract.go

```diff
@@ -911,6 +914,43 @@ func CommitContainerListUpdate(cID interop.Hash256, replicas []uint8) {
 		}
 	}
 
+	// update reports stat after netmap change
```

Comment:
Exactly the thing I want to avoid. Epoch tick should not trigger massive changes in contracts.

Reply:
@roman-khimov, this is not an epoch tick; this is a netmap change, which is done only when some storage nodes leave it. i see no other possibility to drop unused reported values. the previous approach was OK since reports were updated every epoch (an epoch was included in the report's storage key), and left nodes are not included naturally

Reply:
OK, makes some sense.